exercises/brandon_hewer/ex02_oo_basics/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required ( VERSION 2.8.5 )
+
+###############################################################################
+# Setup
+###############################################################################
+# Name
+project ( Shapes )
+
+# Find custom cmake modules
+set ( CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../CMake")
+
+# Common setup
+include ( CommonSetup )
+
+###############################################################################
+# Executable
+###############################################################################
+
+set ( SRC_FILES
+  src/main.cpp
+  src/Circle.cpp
+  src/Square.cpp
+  src/Rectangle.cpp
+  src/Triangle.cpp
+  src/ShapeSorter.cpp
+)
+
+set ( HDR_FILES
+  src/Circle.h
```

Comment:
:+1: for registering the header files (else they won't show up in Visual Studio).
You should put them in a separate folder though, something like `inc`